modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -183,7 +253,34 @@ CREATE DATABASE slow WAIT 5 SECONDS
 
 The `success` column provides an aggregate status of whether or not the command is considered successful.
 Thus, every row has the same value, determined on a successful completion without a timeout.
+======
+[.include-with-cypher-25]
+======
+Sub-clause `WAIT` allows you to specify a time limit in which the command must complete and return.
```

Comment:
```suggestion
Sub-clause `WAIT` allows you to specify a time limit for the command to complete and return.
```

Reply:
This is the same in the Cypher 5 section above also.